src/components/Person/Person.jsx

```diff
@@ -3,20 +3,26 @@
 export const Person = ({ person }) => {
   const { name, age, sex, isMarried, partnerName } = person;
 
+  let statusPartner;
+
+  if (isMarried) {
```

Comment:
You could keep `isMarried` ternary in place - the idea was to get rid of ternary nesting, but one level look good
just FYI 🙂